packages/shaka-video-element/shaka-video-element.js

```diff
@@ -1,4 +1,5 @@
 import { CustomVideoElement } from 'custom-media-element';
+import './server-safe-globals';
```

Comment:
```suggestion
import './server-safe-globals.js';
```

add the extension because browsers don't automatically alias that like node or bundlers...